<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-05-11T11:39:12 ICT" name="Default suite" finished-at="2022-05-11T11:39:40 ICT" duration-ms="28455">
    <groups>
    </groups>
    <test started-at="2022-05-11T11:39:12 ICT" name="Default test" finished-at="2022-05-11T11:39:40 ICT" duration-ms="28455">
      <class name="newTest.NewTest2">
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:newTest.NewTest2@4a22f9e2]" started-at="2022-05-11T11:39:12 ICT" name="beforeTest" finished-at="2022-05-11T11:39:22 ICT" duration-ms="10722" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method signature="demo1()[pri:0, instance:newTest.NewTest2@4a22f9e2]" started-at="2022-05-11T11:39:23 ICT" name="demo1" finished-at="2022-05-11T11:39:26 ICT" duration-ms="3795" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- demo1 -->
        <test-method signature="demo2()[pri:0, instance:newTest.NewTest2@4a22f9e2]" started-at="2022-05-11T11:39:26 ICT" name="demo2" finished-at="2022-05-11T11:39:31 ICT" duration-ms="4832" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Nguyen] but found [Phạm Ngọc Thạch – Wikipedia tiếng Việt]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Nguyen] but found [Phạm Ngọc Thạch – Wikipedia tiếng Việt]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at newTest.NewTest2.demo2(NewTest2.java:47)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- demo2 -->
        <test-method signature="demo3()[pri:0, instance:newTest.NewTest2@4a22f9e2]" started-at="2022-05-11T11:39:31 ICT" name="demo3" finished-at="2022-05-11T11:39:37 ICT" duration-ms="6149" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- demo3 -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:newTest.NewTest2@4a22f9e2]" started-at="2022-05-11T11:39:37 ICT" name="afterTest" finished-at="2022-05-11T11:39:40 ICT" duration-ms="2910" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- newTest.NewTest2 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
